src/main/resources/application.yaml

```diff
@@ -973,9 +973,9 @@ feesAdminOfficerEmailAddress: ${FEES_ADMIN_OFFICER_EMAIL}
 paymentExceptionsAdminOfficerEmailAddress: ${PAYMENT_EXCEPTIONS_ADMIN_OFFICER_EMAIL:payment-exceptions-ao-aat@example.com}
 apcHomeOfficeEmailAddress: ${HOME_OFFICE_EMAIL_APC}
 lartHomeOfficeEmailAddress: ${HOME_OFFICE_EMAIL_LART}
-upperTribunalNoticesEmailAddress: ${IA_UPPER_TRIBUNAL_NOTICES_EMAIL}
-nbcEmailAddress: ${IA_NBC_EMAIL}
-ctscEmailAddress: ${IA_CTSC_EMAIL}
+upperTribunalNoticesEmailAddress: ${IA_UPPER_TRIBUNAL_NOTICES_EMAIL:upper-tribunal-notices@example.com}
```

Comment:
@burale we are no longer allowed to have defaults in this yaml file.

Reply:
Removed